extensions/persistence/org.eclipse.smarthome.persistence.h2sql/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.6"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="java"/>
+	<classpathentry kind="lib" path="lib/h2-1.3.176.jar"/>
```

Comment:
We can, but I am using the most recent stable version. We can use a beta version if you wish, and I don't mind using 1.4.186, but thought that for such a product that may be used commercially, that using beta libraries might be frowned upon?